src/driverDUOstereo.cpp

```diff
@@ -198,8 +198,8 @@ bool DUOStereoDriver::initializeDUO()
 	 * so we have to pass as double, and then cast to float to satisfy DUOResolutionInfo 
 	 * fps parameter requirement of type float
 	 */
-	double 	framesPerSecond;
-	_priv_nh.param("FPS", framesPerSecond, 30.0);
+	int 	framesPerSecond;
```

Comment:
If you look in the documentation https://duo3d.com/docs/articles/api under DUOResolutionInfo they have this as a float.  Also if you look under Enumerate resolution, you pass a float. Under the specs for mini-lx its said that it can do 0.1-3000+ fps